pymysql/converters.py

```diff
@@ -309,13 +309,8 @@ def through(x):
     return x
 
 
-#def convert_bit(b):
-#    b = "\x00" * (8 - len(b)) + b # pad w/ zeroes
-#    return struct.unpack(">Q", b)[0]
-#    
-#     the snippet above is right, but MySQLdb doesn't process bits,
-#     so we shouldn't either
```

Comment:
1) You can manually convert result value.
2) You can override converter mapping.

But I don't recommend (2).  Converter function can't know connection encoding and column encoding.  I think the API is broken.